docs/reference/modules/cluster/misc.asciidoc

```diff
@@ -1,6 +1,12 @@
 [[misc-cluster]]
 === Miscellaneous cluster settings
 
+`cluster.dynamic_name`::
+
+      Assign a name to the cluster that will appear in info requests (e.g.
+      `GET /`).  If unset, the cluster's dynamic name will default to the
```

Comment:
I tend to go with - if not set, we simply don't raturn that field.

Reply:
That is a fair POV - but conflicts with the agreement we already have with cloud.  If you feel this is a bad move I am happy to raise it with them

Reply:
It feels weird to me from an API design. It also bubble up the notion of a dynamic cluster name which feels strange next to cluster name (all due to legacy, but it is what it is). If people feel otherwise about it I'm OK but I'd prefer to have the translation done out of ES. Also people that always set dynamic names don't need to worry about the fallback anyway.